packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -632,11 +632,10 @@ class _RangeSliderState extends State<RangeSlider> with TickerProviderStateMixin
         return Thumb.end;
       }
     } else {
-      // Snap position on the track if its in the inactive range.
-      if (tapValue < values.start || inStartTouchTarget) {
+      // Choose the closest thumb and snap position.
```

Comment:
remember to also update the doc comment of `_defaultRangeThumbSelector` it is still referring to the old behavior